commands/user_test.go

```diff
@@ -870,3 +870,79 @@ func (s *MmctlUnitTestSuite) TestResetUserMfaCmd() {
 		s.Require().Equal(printer.GetErrorLines()[1], "Unable to find user '"+users[3]+"'")
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestListUserCmdF() {
+	s.Run("Listing users with paging", func() {
+		printer.Clean()
+
+		emailArg := "example@example.com"
+		mockUser := model.User{Username: "ExampleUser", Email: emailArg}
+		page := 0
+		perPage := 1
+
+		cmd := &cobra.Command{}
+		cmd.Flags().Int("per-page", perPage, "")
+		cmd.Flags().Int("page", page, "")
+
+		s.client.
```

Comment:
Please return 2 pages of users, each page having 1 user. And verify that both users are returned.